pkg/sidecar/podmutator.go

```diff
@@ -45,7 +45,7 @@ type sidecarPodMutator struct {
 
 var _ webhookhandler.PodMutator = (*sidecarPodMutator)(nil)
 
-func NewMutator(logger logr.Logger, config config.Config, client client.Client) *sidecarPodMutator {
+func NewMutator(logger logr.Logger, config config.Config, client client.Client) *sidecarPodMutator { //nolint:revive
```

Comment:
```suggestion
func NewMutator(logger logr.Logger, config config.Config, client client.Client) *sidecarPodMutator {
```

Nitpick: this should be added in the same PR with revive